other/get-up-tweet.js

```diff
@@ -6,6 +6,8 @@ import {displayListify, sortPeople} from '../shared/utils'
 import shortenEpisodeUrl from './shorten-episode-url/logic'
 import episodeList from './utils/episode-list'
 
+export {validateMessageAndAddEmojiIfOk}
```

Comment:
Sure thing! I had to move `.eslintrc` over as well. Not sure if there's a way to specify that `console: 0` rule for just the one function but let me know if you'd prefer something different.